_config.yml

```diff
@@ -30,7 +30,7 @@ kind: "lesson"
 repository: <USERNAME>/<PROJECT>
 
 # Email address, no mailto:
-email: "team@carpentries.org"
+email: "team@esmvaltool.org" # What should we use here?
```

Comment:
@bjoernbroetz @hb326 Are you involved with setting up the user engagement team email address?

Reply:
@bouweandela: we have not thought that far ahead with the user engagement team setup, I am afraid. I think the user engagement email (general contact) would be the right choice to put in here, but at the moment this email address does not exist yet.

Reply:
@hb326 Any news on the user engament team yet? Do you have an email address now?

Reply:
The email address in contributing.md needs to be updated as well.